.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,106 @@
+name: Report a bug
+description: Errors and regression reports with complete reproducing test cases.
+labels: ["status:triage", "bug"]
+body:
+  - type: markdown
+    attributes:
+      value: >
+        Bug reports aren't usually really **🐛 bugs**. More often than not they are questions about how to use the product.
+        Hit us up on our [Community Slack](https://pytorch-lightning.slack.com) first if that is the case.
+        You're probably going to get a faster response and you're going to learn about using the product.
+
+  - type: markdown
+    attributes:
+      value: >
+        If you are new to making bug reports, here are a few guidelines for you:
+
+        1. A clear and concise description of the bug - keep it to a line or two
+
+        2. A clear and concise code example that reproduces the problem. To get that use the [BoringModel] (https://colab.research.google.com/github/Lightning-AI/lightning/blob/master/examples/pl_bug_report/bug_report_model.ipynb)!
```

Comment:
Boring model applies only to trainer bugs. What should we do for apps? @manskx @hhsecond

Reply:
Removed this